src/index.ts

```diff
@@ -21,11 +21,18 @@ export const beautifier: Beautifier = {
     {
       type: DependencyType.Node,
       name: "ESLint",
-      package: "eslint",
+      package: "eslint"
```

Comment:
Put a comma at the end here

Reply:
Can we enable a linter for this? I really don't know when to put a comma and when I shouldn't …

Reply:
https://palantir.github.io/tslint/rules/trailing-comma/ should be catching it, but it's not.

Reply:
Looks like it is set to `ignore` for `multiline`: https://github.com/Unibeautify/beautifier-eslint/blob/master/tslint.json#L294-L300

@muuvmuuv Could you tweak this and make sure TSLint is telling you to add the comma? :)